datanode/src/main/java/com/odysseusinc/arachne/datanode/model/analysis/Analysis.java

```diff
@@ -76,6 +76,14 @@ public class Analysis {
     @NotNull
     @Column(name = "analysis_folder")
     private String analysisFolder;
+    @ManyToOne
+    @JoinColumn(name = "initial_state_id")
+    private AnalysisStateEntry initialState;
```

Comment:
Defect: I don't see the reference to AnalysisStateEntry as necessary here.
If we need to store the start timestamp convenientrly, let's just add field for that

Reply:
I don’t see much of a difference here, to be honest. The data will end up being duplicated, and it’s just a simple join.